ibc-apps/ics721-nft-transfer/types/src/events.rs

```diff
@@ -1 +1,209 @@
 //! Defines Non-Fungible Token Transfer (ICS-721) event types.
+use ibc_core::channel::types::acknowledgement::AcknowledgementStatus;
+use ibc_core::primitives::prelude::*;
+use ibc_core::primitives::Signer;
+use ibc_core::router::types::event::ModuleEvent;
+
+use super::Memo;
+use crate::{PrefixedClassId, TokenIds, MODULE_ID_STR};
+
+const EVENT_TYPE_PACKET: &str = "nft_packet";
```

Comment:
```suggestion
const EVENT_TYPE_PACKET: &str = "non_fungible_token_packet";
```

Get it consistent with the ICS-20 `EVENT_TYPE_PACKET` and with the ref: https://github.com/bianjieai/nft-transfer/blob/b57ce734069ee36e8024fa480111e115e959138e/types/events.go#L6